src/Forms/GridField/GridFieldFooter.php

```diff
@@ -66,7 +66,7 @@ public function getHTMLFragments($gridField)
             'footer' => $forTemplate->renderWith(
                 $template,
                 [
-                    'Colspan' => count($gridField->getColumns() ?? [])
+                    'Colspan' => $gridField->getColumnCount(),
```

Comment:
This used to give a different result than `getColumnCount()`.

Now that I've fixed `getColumnCount()` it's the same result, but this change protects us from discrepancies resulting from any further changes in that method.